Rock/Workflow/Action/Communications/SendNotification.cs

```diff
@@ -216,13 +217,34 @@ public override bool Execute( RockContext rockContext, WorkflowAction action, Ob
             }
             sound = sound.AsBoolean() ? "default" : "";
 
+            string url = GetAttributeValue(action, "Url");
+            Guid urlGuid = url.AsGuid();
+            if (!urlGuid.IsEmpty())
+            {
+                var attribute = AttributeCache.Get(urlGuid, rockContext);
+                if (attribute != null)
+                {
+                    string urlAttributeValue = action.GetWorkflowAttributeValue(urlGuid);
+                    if (!string.IsNullOrWhiteSpace(urlAttributeValue))
+                    {
+                        if (attribute.FieldType.Class == "Rock.Field.Types.BooleanFieldType")
```

Comment:
@richarddubay can you help me understand why the code is checking that the attribute.FieldType.Class is equal to BooleanFieldType?

Reply:
Yep, it's doing that because I totally typed the wrong thing. 🤦‍♂️ It should be `Rock.Field.Types.TextFieldType`. That's a good catch! I'll fix it.